lib/data/wrapper.js

```diff
@@ -6,6 +6,7 @@ import inMemory from './in_memory/backend';
 import config from '../Config';
 import MD5Sum from '../utilities/MD5Sum';
 import assert from 'assert';
+import DpClient from './dpa/dpClient';
```

Comment:
For now it calls individual backend clients. Our next purpose is to store fragments in multiple backend clients